_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+
+Today, Knowledge Shares are for more than just engineers. They are for product managers, designers, data analysts, and other team stakeholders. Sometimes engineers from other teams even join my team's Knowledge Shares, but scheduling becomes a hassle at a certain scale. We start each Knowledge Share with team-wide topics that apply to more than only engineers; we then move on to engineering-specific topics. This lets non-engineers still contribute while also letting them drop off so the engineers can dig into code.
+
+Here are some examples of things that we have used Knowledge Shares for:
+
+- Sharing early designs for upcoming project work.
+- Going over work-in-progress pull requests and soliciting feedback on the technical approach.
+- Spiking on big-picture projects together, to help us plan what work needs to happen and in which order.
+- Exploring anonymized user sessions to learn more about how our product actually gets used in the real world.
+- Investigating small quality-of-life problems with our codebase's developer experience and working on solutions.
+- Brainstorming about how to address specific pieces of technical debt.
+
+Topics range from the technical to the product-focused, from what has already shipped to what is still on the drawing board. This breadth of focus helps everyone on the team feel like they can contribute. We keep a shared document of evergreen topic ideas and everyone is encouraged to add to it. As Tech Lead, I also send Slack reminders before each meeting to solicit for timely topics.
+
+Throughout the normal course of our week, topics will naturally come up for discussion – topics that would normally need their own meeting to be scheduled. But instead, we can use the time we already have scheduled. For example: if an engineer has feedback on a new feature's design, we don't need to schedule a dedicated meeting for that conversation – we can use a Knowledge Share. It might feel like having recurring "Knowledge Share" meetings only adds more meetings to everyone's already-busy calendars. But in our experience, these Knowledge Shares prevent us from scheduling _yet more_ one-off meetings, so it balances out as a time-saver
+
+One trick I like to pull with Knowledge Shares is to use them to encourage the team's understanding that _learning_ is our paramount goal. That is to say, our goal isn't to _build_ some new feature, but is rather to _learn **how** to build_ that new feature. That's why I emphasize in our Knowledge Shares that _learning is contributing_ (language which I added to our engineering onboarding docs). New team members might not have topics of their own, but they're encouraged to contribute by learning and by asking questions.
```

Comment:
Non-blocking: "One trick I like to pull" sounds slightly devious and manipulative to me. Is there a way to rephrase this to sound more like you aren't trying to fool us? 

Maybe something like: 

```suggestion
Knowledge Shares are a great opportunity to emphasize that _learning_ is our team's paramount goal. That is to say, our goal isn't to _build_ some new feature, but is rather to _learn **how** to build_ that new feature. That's why I emphasize in our Knowledge Shares that _learning is contributing_ (language which I added to our engineering onboarding docs). New team members might not have topics of their own, but they're encouraged to contribute by learning and by asking questions.
```

Reply:
Also: "language which I added to our engineering onboarding docs" is a good opportunity to link to our onboarding docs.

Reply:
Sadly, that would be in the potential repo and is consequently closed-source. (The docs in question are a GitHub issue template for onboarding, and issues need to be closed, so our hands are tied a bit here.)